anycloud/cli/alan/anycloud.ln

```diff
@@ -158,3 +133,29 @@ fn toString(reqBody: ServiceRequestBody): string {
   const messageStr = '"message": "' + reqBody.message + '"';
   return '{' + errorCodeStr + ', ' + alanVersionStr + ', ' + levelStr + ', ' + messageStr + '}';
 }
+
+fn getCleanError(error: string): string {
+  return error
+    .split("\n").join(" ")
+    .split("").map(fn(value: string): string {
+      return value == '"' ? "'" : value;
```

Comment:
Maybe replacing `"` with `\"` would be better?

Reply:
Yes, actually would be thee proper json serialization of the string. will try it